keyboards/dumbpad/v1x_oled/keymaps/default/keymap.c

```diff
@@ -0,0 +1,207 @@
+/* Copyright 2020 imchipwood
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#include <stdio.h>
+char wpm_str[10];
```

Comment:
```suggestion
```

Reply:
This is needed for the ```oled_task_user()``` function

Reply:
Moved out of keymap in https://github.com/qmk/qmk_firmware/pull/17058/commits/165318675c78d8f4697645e237d2aff5672aa12d